[REVIEW] Update RAPIDS Demo webpage (#191)
rapidsai/docs: demos/rapids-demo.md

> To get the latest RAPIDS version of a specific platform combination, simply exclude the RAPIDS version.  For example, to pull the latest version of RAPIDS for the `runtime` image with support for CUDA {{ CUDA_VER }}, Python {{ PYTHON_VER }}, and Ubuntu 18.04, use the following tag:

Reviewer: ```suggestion
To get the latest RAPIDS version of a specific platform combination, simply exclude the RAPIDS version.  For example, to pull the latest version of RAPIDS for the `runtime` image with support for CUDA {{ CUDA_VER }}, Python {{ PYTHON_VER }}, and Ubuntu {{ UBUNTU_VER }}, use the following tag:
```

Reviewer: Can you add this change to your PR?

```

> {{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}

Reviewer: ```suggestion
cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
```
This change contradicts the line above it, which states that the latest RAPIDS images can be obtained by omitting the RAPIDS version from the image name. Please revert.

```

> Many users do not need a specific platform combination but would like to ensure they're getting the latest version of RAPIDS, so as an additional convenience, a tag named simply `latest` is also provided which is equivalent to `{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-ubuntu18.04-py{{ PYTHON_VER }}`.

Reviewer: ```suggestion
Many users do not need a specific platform combination but would like to ensure they're getting the latest version of RAPIDS, so as an additional convenience, a tag named simply `latest` is also provided which is equivalent to `{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-ubuntu{{ UBUNTU_VER }}-py{{ PYTHON_VER }}`.
```
we should probably make the Ubuntu version a variable like the others here.

Author: agreed!
